Add Clarification to Requirements 2 (#2)
willcodefortea/cake-shop-kata: requirements_2.md

## Examples

If a cake order is received in the morning (ie, before 12pm) then baking starts on the same day.

> Cakes not ordered before 12pm are started the following day.

Reviewer: ```suggestion
Cakes ordered after 12pm are started the following day.
```

Author: Considered this originally. Reason I don't like this wording is because it leaves open to interpretation the case for exactly 12pm. Not saying there isn't a better way of wording it than I have, but wanted to close this loophole.
